batch/sc-batch-fargate.yaml

```diff
@@ -57,30 +57,30 @@ Parameters:
       - true
       - false
   Command:
-    Type: List<String>
+    Type: String
     Description: >
-      The list of commands to pass to the container image (i.e. ["echo", "hello", "world"])
+      A pipe seperated list of commands to pass to the container image (i.e. echo|hello|world)
     ConstraintDescription: >
-      Must be a list of commands
+      Must be a pipe delimited string of commands
```

Comment:
As per my above (nick-picky) comment, 'Must be a pipe delimited string.'

Reply:
done